src/fsharp/FSharp.Compiler.Private/FSharp.Compiler.Private.fsproj

```diff
@@ -41,7 +41,6 @@
   <ItemGroup>
     <InternalsVisibleTo Include="fsc" />
     <InternalsVisibleTo Include="FSharp.Build" />
-    <InternalsVisibleTo Include="fsi" />
     <InternalsVisibleTo Include="fsiAnyCpu" />
```

Comment:
And fsi any cpu? … it is also fsi.

Reply:
I was planning on doing one at a time, but I'll look into this and see if it's not too big of a change.